src/main/scala/no/ndla/conceptapi/controller/ConceptController.scala

```diff
@@ -310,6 +311,39 @@ trait ConceptController {
       }
     }
 
+    get(
+      "/tag-search/",
+      operation(
+        apiOperation[TagsSearchResult]("getTags-paginated")
+          summary "Retrieves a list of all previously used tags in concepts"
+          description "Retrieves a list of all previously used tags in concepts"
+          parameters (
+            asHeaderParam(correlationId),
+            asQueryParam(query),
+            asQueryParam(pageSize),
+            asQueryParam(pageNo),
+            asQueryParam(language)
+        )
+          responseMessages (response403, response500)
+          authorizations "oauth2")
+    ) {
+      val userInfo = user.getUser
+      doOrAccessDenied(userInfo.canWrite) {
```

Comment:
Trenger man skriverettigheter for å hente ut dette?

Reply:
Gikk litt for fort i svingen der, nei det skal ikke være nødvendig med authorization på dette endepunktet